src/06-positron-rag/lib/insert/prepareArticlesCollection.ts

```diff
@@ -79,16 +79,16 @@ export async function prepareArticlesCollection() {
           },
         },
       },
-      // {
-      //   name: "keyword",
-      //   dataType: ["text"],
-      //   moduleConfig: {
-      //     "text2vec-openai": {
-      //       skip: true,
-      //       // vectorizePropertyName: false,
-      //     },
-      //   },
-      // },
+      {
+        name: "keywords",
+        dataType: ["text[]"],
```

Comment:
(Tangential, but I had to sort out the syntax for an array-valued text field here, so now I can uncomment this)